configs/twilio_paste.json

```diff
@@ -1,27 +1,88 @@
 {
   "index_name": "twilio_paste",
   "start_urls": [
-    "https://paste.twilio.design/"
+    {
+      "url": "https://paste.twilio.design/"
+    },
```

Comment:
```suggestion
```
The first `start_url` matching applies. This URL will always match and the other item will have no effect.